swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+
+    def __init__(self, condition = 0):
+        super().__init__(condition = condition, category = "Clothing")
+        self.condition = condition
+        
+        
+        
+    def __str__(self):
+        self = "The finest clothing you could wear."
+        return f"{self}"
```

Comment:
As with the parent implementation in `Item`, here, we should _not_ re-assign `self`. We can instead return the desired string directly.

```python
    def __str__(self):
        return "The finest clothing you could wear."
```